scripts/profiler/rtt_nordic_profiler_host.py

```diff
@@ -3,13 +3,10 @@
 #
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 
-from pynrfjprog import API
+from pynrfjprog.LowLevel import API
+from pynrfjprog.APIError import APIError
```

Comment:
I see this script imports from `pynrfjprog` but `pynrfjprog` is not not automatically being installed by users.

Should we add `pynrfjprog` to one of the following files ?
https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/master/scripts/requirements.txt
https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/master/scripts/requirements-extra.txt

Reply:
It is already added as part of `requirements.txt` in `profiler` directory.
I think its better that way, because you do not need these dependencies if you are not going to use profiler.